recipes-wpe/wpeframework/wpeframework-plugins_git.bbappend

```diff
@@ -1,15 +1,13 @@
-
-WPE_COMPOSITOR = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'compositor_wayland', 'compositor_rpi', d)}"
-
 PACKAGECONFIG_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'compositor', '${WPE_COMPOSITOR}', '', d)} \
-                        displayinfo playerinfo"
-
+                        dialserver dial_youtube displayinfo playerinfo power"
 # snapshot implemented with userland support, not applicable on vc4graphics
 PACKAGECONFIG_append = " ${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', '', 'snapshot', d)}"
 
-WPE_PLAYERINFO_DEP     = "gstreamer1.0"
-WPE_SNAPSHOT_DEP       = "${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', '', 'userland', d)}"
-WPE_WIFICONTROL_DEP    = "linux-firmware-bcm43430"
+# Additional config added for testing
+PACKAGECONFIG_append = " filetransfer inputswitch languageadministrator processmonitor resourcemonitor svalbard systemcommands webproxy webshell"
 
-# Raspberry PI 3 has 4 cores, set to 4 for RPIs
-WEBKITBROWSER_THREADEDPAINTING = "2"
+WPE_COBALT_STARBOARD_CONFIGURATION_INCLUDE = "third_party/starboard/wpe/rpi/configuration_public.h"
```

Comment:
Not sure about those couple of lines yet. Let's leave it for now

Reply:
Ok, it is for cobalt header dependency with plugin